### ISSUE TITLE
Not storing other fields with repeater

### ISSUE THREAD

**Issue by mainpart** [nk-o/lazy-blocks#116]
I'm trying to make block containing mix of repeaters and other fields (say textarea's)
When i'm trying to output textarea in handlebars result is null because no content other than repeater is stored in post. 
Post content is
`<!-- wp:slug {"repeater1":"%5B%5D","repeater2":"%5B%,..."blockId":"...","blockUniqueClass":"..."} /-->`

All other fields are lost.

**Comment by mainpart** [comment]
worth to mention i'm relatively new to block editor and this is my first attempt with guttenberg so i'm not sure where shoud they be stored

**Comment by nk-o** [comment]
Hi.

I have tested and it is working fine for me.

1. ![image](https://user-images.githubusercontent.com/14810891/74375929-4992c000-4df2-11ea-9c40-2b751cd1261a.png)
2. ![image](https://user-images.githubusercontent.com/14810891/74375892-35e75980-4df2-11ea-8b03-c06715501bda.png)

Exported block:
[lzb-export-blocks-2020-02-12.zip](https://github.com/nk-o/lazy-blocks/files/4194968/lzb-export-blocks-2020-02-12.zip)

**Comment by mainpart** [comment]
upgraded to latest. 
still not working for me.

[lzb-export-blocks-2020-02-18.zip](https://github.com/nk-o/lazy-blocks/files/4218704/lzb-export-blocks-2020-02-18.zip)

![image](https://user-images.githubusercontent.com/2713266/74728705-d49e0b00-5275-11ea-8c51-ca8303a017c5.png)

Result with the_content() is empty
![image](https://user-images.githubusercontent.com/2713266/74728607-a9b3b700-5275-11ea-8c95-1c11c9243450.png)

**Comment by mainpart** [comment]
saved post_content from table is:

`<!-- wp:lazyblock/expert {"blockId":"11BImb","blockUniqueClass":"lazyblock-expert-11BImb"} /-->`

related postmeta is ok because i've checked each item to duplicate in postmeta. post content not stored. where should i dig the issue?

**Comment by mainpart** [comment]
Also when saving i can see a bunch of empty metas in payload. They came from other blocks that totally unrelated to current page and they are not presenting on the page.

![image](https://user-images.githubusercontent.com/2713266/74730923-a02c4e00-5279-11ea-9451-b46225880723.png)

![image](https://user-images.githubusercontent.com/2713266/74730976-b934ff00-5279-11ea-8e3c-c9f1b85b6d39.png)

**Comment by mainpart** [comment]
i think i got it. When we are not saving in meta we saving in body.
It's a mutual exclusive options. 
I think they should not be - we can use meta values for some functional wp_query variables and output in the handlebars template as well. 

For example, we can  have a custom post type "person" with "age" variable and we want this post tybe to be queryable and sortable by that meta field.

**Comment by nk-o** [comment]
Once you decided to use "Save in Meta" option, you need to register it in WordPress. This is why you should use a unique prefix in the meta name to prevent conflicts.

Unfortunately, this meta value is not available as a block attribute. This is how it was designed in Gutenberg.

**Comment by mainpart** [comment]
my question was about:
can we store block in meta value and in body (gutenber block) simultaneously? 

For example i'm creating block and want save it to meta and use it in {{handlebars}} template as well. For now i can do ether one or another.

**Comment by nk-o** [comment]
Nope, you can't, this is how meta attributes designed in Gutenberg blocks.
